csi/crypto/crypto.go

```diff
@@ -95,9 +105,9 @@ func EncryptVolume(devicePath, passphrase string, cryptoParams *EncryptParams) e
 }
 
 // OpenVolume opens volume so that it can be used by the client.
-func OpenVolume(volume, devicePath, passphrase string) error {
-	if isOpen, _ := IsDeviceOpen(VolumeMapper(volume)); isOpen {
-		logrus.Infof("Device %s is already opened at %s", devicePath, VolumeMapper(volume))
+func OpenVolume(volume, dataEngine, devicePath, passphrase string) error {
```

Comment:
What is this `devicePath`? the original volume device path `/dev/longhorn/<volume name>` or the encrypted device path `VolumeMapper(volume, dataEngine)`?

Reply:
`devicePath` will be the original volume device path`/dev/longhorn/<volume-name>`

Reply:
NIT: Plz change the variable name or add a comment to avoid confusion.

Reply:
Modified the variable name and added a comment. Thanks